client-mvp-04/src/sass/elements/_inputs.scss

```diff
@@ -42,3 +42,43 @@
     top: 14px;
   }
 }
+
+.radio-button-container {
+  display: inline-flex;
```

Comment:
Could you arrange radio-buttons vertically?

Reply:
Okay, I will.

Reply:
Actually changing the display to other than inline will break the button. Actual radio button is inlined with the custom one so that custom radio button appear as an actual radio button. To arrange vertically have to wrap the radio button with a block level element. 
```css
So, Do I have to arrange the button vertically using block level element in uiKit.js???
```

Reply:
Sure, you can wrap the radio button with a block element

Reply:
Do I need to add any margin around the radio button?

Reply:
Do I need to make 22px border to the bottom of the radio button like this?
![image](https://user-images.githubusercontent.com/43095489/107850738-d241a300-6e2e-11eb-9e7e-fbcbbd9b7e71.png)
![image](https://user-images.githubusercontent.com/43095489/107850788-42502900-6e2f-11eb-906f-a365228eb66d.png)

Reply:
Sure, add 22px for margin-bottom.